lib/models/PostGram.js

```diff
@@ -27,4 +28,24 @@ module.exports = class PostGram {
       return rows.map(row => new PostGram(row));
     }
 
+    static async findById(id) {
```

Comment:
very close!   In order to fulfill the rest of this, you'll need to join to your Users table as well by id.